homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/games.html

```diff
@@ -16,12 +18,52 @@
                             <a href="./games.html">Games</a>
                         </li>
                         <li class="nav_item">
-                            <a href="./about.html">About</a>
+                            <a href="./index.html">About</a>
+                        </li>
+                        <li class="nav_item">
+                            <a href="./login-page.html">Login</a>
                         </li>
                     </ul>
                 </nav>
             </div>
         </header>
+        <section class="filter">
+            <form action="" class="filter_form">
+                <div class="filter_drop_down">
+                    <select name="genre" class="genre_list" id="genre">
+                        <option value="genre" selected hidden disabled>Genre</option>
+                        <option value="sport">Sport</option>
+                        <option value="shooter">Shooter</option>
+                        <option value="racing">Racing</option>
+                        <option value="horror">Horror</option>
+                    </select>
+                </div>
+                <div class="filter_radio">
+                    <label for="" class="platform">
```

Comment:
Please implement acceptance criteria from the [task](https://github.com/MastersAcademy/fe-course-basic-2023/issues/137#issue-1955947368)
> По кліку на підпис інпута, він має отримувати фокус(поле для вводу чогось), або змінювати стан (чекбокс). Використовуйте для цього for або <label><input></label>

Reply:
[`for`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#for) should pint to `id` of `input`
This behavior still doesn't work
> По кліку на підпис інпута, він має отримувати фокус(поле для вводу чогось), або змінювати стан (чекбокс). Використовуйте для цього for або

Reply:
I should definitely re-read the "For" topic. This time it is okay, I guess